source/Weapon.cpp

```diff
@@ -155,6 +155,8 @@ void Weapon::LoadWeapon(const DataNode &node)
 				inaccuracy = value;
 			else if(key == "turret turn")
 				turretTurn = value;
+			else if(key == "arc")
+				arc = max(0., value);
```

Comment:
This is not really arc, it feels more like `maximum angle` (`maxAngle` for the variable), to me since we have to accept the minimum one is 0, and just set the max here.

Reply:
I also meant changing the keyword for outfit definition itself though that's less important

Btw why is this one just about a max angle and the other one about a min and max angle? Is it because it was intended to be for the left and right angle?